pulsar-client-cpp/include/pulsar/c/consumer_configuration.h

```diff
@@ -44,6 +46,17 @@ typedef enum {
     pulsar_ConsumerFailover
 } pulsar_consumer_type;
 
+typedef enum {
+    /**
+     * the latest position which means the start consuming position will be the last message
+     */
+    multi_topic_latest,
+    /**
+     * the earliest position which means the start consuming position will be the first message
+     */
+    multi_topic_earliest
+} multi_topic_position;
```

Comment:
This is not really tied with the "multi-topics" features, rather just on where to initialize a newly created subscription